src/components/ItemList.js

```diff
@@ -0,0 +1,21 @@
+import icoHeart from "../img/ic_heart.svg";
+
+export function ItemList({ item }) {
+  return (
+    <>
+      <div className="itemlist-img-wrap">
+        <a href="" className="link">
+          <img src={item.images[0]} alt={item.name + " 이미지"} className="itemlist-img"/>
+        </a>
+      </div>
+      <div className="itemlist-content">
+        <h2 className="itemlist__title"><a href="#" className="link">{item.description}</a></h2>
+        <p className="itemlist__price">{item.price.toLocaleString()}원</p>
```

Comment:
jsx는 항상 깔끔하게 보여지게끔 유지해주시면 좋답니다
예를 들어 이렇게 인라인으로 메서드 써주신 부분들도 
const price = item.price.toLocaleString();

```suggestion
        <p className="itemlist__price">{price}원</p>
```

이런식으로 깔끔하게 정리해주시는 습관 들여보시면 좋을것같아요 :)